docs/en/observability/cloud-monitoring/aws/monitor-aws-firehose.asciidoc

```diff
@@ -0,0 +1,86 @@
+
+[discrete]
+[[aws-elastic-firehose-what-you-learn]]
+=== What you'll learn
+
+In this tutorial, you'll learn how to:
+
+- Install AWS integration in {kib}
+- Create a delivery stream in Amazon Data Firehose
+- Specify the destination settings for your Firehose stream
+- Send data to the Firehose delivery stream
+
+[discrete]
+[[aws-elastic-firehose-before-you-begin]]
+=== Before you begin
+
+Create a deployment using our hosted {ess} on {ess-trial}[{ecloud}].
+The deployment includes an {es} cluster for storing and searching your data, and {kib} for visualizing and managing your data. You also need an AWS account with permissions to pull the necessary data from AWS.
+
+[discrete]
+[[firehose-step-one]]
+=== Step 1: Install AWS integration in {kib}
+
+. Install AWS integrations to load index templates, ingest pipelines, and dashboards into {kib}. In {kib}, navigate to *Management* > *Integrations* in the sidebar. Find the AWS Integration by browsing the catalog.
+
+. Navigate to the *Settings* tab and click *Install AWS assets*. Confirm by clicking *Install AWS* in the popup.
+
+. Install AWS Firehose integration assets in Kibana. 
+
+NOTE: Firehose integration is currently in beta. Make sure to enable *Display beta integrations*.
+
+[discrete]
+[[firehose-step-two]]
+=== Step 2: Create a delivery stream in Amazon Data Firehose
+
+. Go to the https://console.aws.amazon.com/[AWS console] and navigate to Amazon Data Firehose.  
+
+. Click *Create Firehose stream* and choose the source and destination of your Firehose stream. Unless you are streaming data from Kinesis Data Streams, set source to `Direct PUT` and destination to `Elastic`. 
+
+. Provide a meaningful *Firehose stream name* that will allow you to identify this delivery stream later.
+
+NOTE: For advanced use cases, source records can be transformed by invoking a custom Lambda function. When using Elastic integrations, this should not be required.
+
+[discrete]
+[[firehose-step-three]]
+=== Step 3: Specify the destination settings for your Firehose stream
+
+. From the *Destination settings* panel, specify the following settings:
++
+* *Elastic endpoint URL*: Enter the Elastic endpoint URL to point to your Elasticsearch cluster. To find the Elasticsearch endpoint, go to the Elastic Cloud console and select *Connection details*. Here is an example of how it looks like: `https://my-deployment.es.us-east-1.aws.elastic-cloud.com`.
++
+* *API key*: Enter the encoded Elastic API key that you can create in {kib}. To create an API, go to the Elastic Cloud console, select *Connection details* and click *Create and manage API keys*. If you are using an API key with *Restrict privileges*, make sure to review the Indices privileges to provide at least "auto_configure" & "write" permissions for the indices you will be using with this delivery stream. 
```

Comment:
Is  `that you can create in {kib}` necessary? The step talks about using the console, not Kibana.

```suggestion
* *API key*: Enter the encoded Elastic API key. To create an API key, go to the Elastic Cloud console, select *Connection details* and click *Create and manage API keys*. If you are using an API key with *Restrict privileges*, make sure to review the Indices privileges to provide at least "auto_configure" & "write" permissions for the indices you will be using with this delivery stream. 
```